ttnn/cpp/ttnn/tensor/tensor.hpp

```diff
@@ -28,6 +28,11 @@ namespace tt {
 
 namespace tt_metal {
 
+
+namespace distributed {
+    class MeshDevice;
+}
+using MeshDevice = tt::tt_metal::distributed::MeshDevice;
```

Comment:
I am not sure if it is a good idea to call using in so highlevel place near tensor.
Probably moving this using inside of the tensor at least would help.

Reply:
good point, I've removed the `using` in `tensor.hpp` all together. thanks!